Projects/DbCLI/CreateDb.py

```diff
@@ -431,11 +435,11 @@ def CreateDb(options, dataFilename='', supplier=None):
 
 
 if __name__ == '__main__':
-  options, args = parser.parse_args()
+  options = parser.parse_args()
   if options.loadMols:
-    if len(args) != 1:
+    if len(options) != 1:
       parser.error('please provide a filename argument')
-    dataFilename = args[0]
+    dataFilename = options[0]
```

Comment:
These three lines will need to change when you add the `add_argument` for the positional argument.